### DISCUSSION TITLE
No Data - Monitoring

### DISCUSSION THREAD

**Discussion by jvxis** [onemorebsmith/kaspa-stratum-bridge#39] [Q&A] [answered]
Hi, @onemorebsmith. I'm running the monitoring, and no data so far.
![image](https://user-images.githubusercontent.com/108929149/192862458-b72cb289-f344-47a8-b139-86aee19c1a80.png)

Is any additional configuration needed? Do I need to open port 9090?

Thanks,
JVX

**Comment by jvxis** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/108929149/192901308-203dbe69-e689-4425-91ba-aa7a3832dd11.png)
Hi, @onemorebsmith I changed the docker.internal address to pool URL, and it is working.

**Reply by onemorebsmith** [reply]
Ha nice. Looking good!
